<launch>
	<node pkg="dynamic_occpuancy_map" name="map_sim_example" type="map_sim_example" output="screen" >
		<!-- <param name="filter_radius" value="1.0" type="double"/> -->
	<remap from="/my_map/future_status" to="/map_generator/global_cloud"/>
	<!-- <remap from="/my_map/cloud_ob" to="/map_generator/global_cloud"/> -->
	</node>
	
	<!-- <node pkg="rviz" type="rviz" name="rviz3" args="-d $(find dynamic_occpuancy_map)/rviz/original_pointcloud.rviz"/> -->
	<!-- <node pkg="rviz" type="rviz" name="rviz2" args="-d $(find dynamic_occpuancy_map)/rviz/future_status.rviz"/> -->
	<node pkg="rviz" type="rviz" name="rviz1" args="-d $(find dynamic_occpuancy_map)/rviz/boxes.rviz" />
	
	<!-- 数据包的frame_id是"map" -->
	<!-- <arg name="bag_dir" default="$(find dynamic_occpuancy_map)/data"/>
	<arg name="dataset_name" default="street"/>
	<node pkg="rosbag" type="play" name="player1" args="$(arg bag_dir)/$(arg dataset_name).bag"/> -->
</launch>
